src/unix/linux_like/android/b64/aarch64/mod.rs

```diff
@@ -55,6 +55,12 @@ s! {
         pub pc: u64,
         pub pstate: u64,
     }
+
+    pub struct user_fpsimd_struct {
+        pub vregs: [__uint128_t; 32],
```

Comment:
This type needs to be imported:
```suggestion
        pub vregs: [::__uint128_t; 32],
```